config/application.rb

```diff
@@ -1,38 +1,31 @@
 require_relative "boot"
 
-# only require Rails parts that we actually use, this shaves off some memory
-# ActiveStorage, ActionCable and TestUnit are not currently used by the app
-# see <https://github.com/rails/rails/blob/v5.2.3/railties/lib/rails/all.rb>
-%w[
-  active_record/railtie
-  action_controller/railtie
-  action_view/railtie
-  action_mailer/railtie
-  active_job/railtie
-  sprockets/railtie
-].each do |railtie|
-  require railtie
-end
+require "rails"
+# Pick the frameworks you want:
+require "active_model/railtie"
+require "active_job/railtie"
+require "active_record/railtie"
+require "active_storage/engine"
```

Comment:
we can comment this as we don't need it, yet :D less memory used in the meantime